cime/src/components/data_comps/docn/cime_config/config_component.xml

```diff
@@ -30,6 +30,7 @@
     <desc option="AQP8">  analytic aquaplanet sst - option 8</desc>
     <desc option="AQP9">  analytic aquaplanet sst - option 9</desc>
     <desc option="AQP10">  analytic aquaplanet sst - option 10</desc>
+    <desc option="AQP11">  CONSTANT, UNIFORM aquaplanet sst - option 11</desc>
```

Comment:
So here's one place where my PR duplicates this functionality. I don't mean to toot my own horn, but I like my implementation better. I added an "AQPCONST" mode. I think more descriptive names are better. But since the names aren't different I don't think there would be a conflict in this file.

Reply:
Well the convention in docn is to just number the options so you should stick with that or change all the option names to be descriptive.

Reply:
The numbers come from the original aqua planet MIP protocol, so the original numbers make sense, therefore making 11 = constant SST is not consistent with the previous options and a more description option makes way more sense, like AQPFILE

Reply:
that's a good point.  Please open an issue in ESMCI/cime.